README.md

```diff
@@ -113,49 +81,74 @@ through the Docker Compose
 docker-compose build
 ```
 
-### Start
+# How to start the request editor
```

Comment:
Request Editor -- worth doing a cmd+f to find all instances of lower case and make them all consistent uppercase